e2e/mobile/mobile-learn.spec.ts

```diff
@@ -31,7 +31,22 @@ const publicSB = orderedSuperBlockInfo
   .filter(sb => sb.public === true && !nonEditorSB.includes(sb.dashedName))
   .map(sb => sb.dashedName);
 
-const typedCurriculum = currData as Curriculum;
+const incompatible = [
+  'certifications',
+  'a2-english-for-developers',
+  'b1-english-for-developers'
+];
+
+const removeNonCompatibleSuperblocks = (currData: Curriculum): Curriculum => {
+  const copy = currData;
+
+  for (const superBlock of incompatible) {
+    delete copy[superBlock];
+  }
+  return copy;
+};
+
+const typedCurriculum = removeNonCompatibleSuperblocks(currData as Curriculum);
```

Comment:
```suggestion
const typedCurriculum = removeNonCompatibleSuperblocks(currData as never);
```

Fine then we'll extra cheat it.